src/shared/types/index.js

```diff
@@ -66,6 +66,7 @@ export const apiFilterType = PropTypes.shape({
     created_after: PropTypes.string,
     created_before: PropTypes.string,
     feedback: PropTypes.string,
+    punctuality: PropTypes.string,
```

Comment:
As I can see, in the backend they use `deadline` for this. Can we use that as well? Punctuality let me think at something else :-).

Reply:
The backend has named this filter punctuality so I don't understand where deadline would come into play. Can you clarify?

Reply:
I see. At category level they call this deadline 
![image](https://user-images.githubusercontent.com/24289432/111602772-fb5da680-87d3-11eb-8554-cc1ad8b1988d.png)